ridemyway/api/vehicle/models.py

```diff
@@ -0,0 +1,16 @@
+from django.conf import settings
+from django.db import models
+
+from ridemyway.api.utilities.base_classes.models import BaseModel
+
+
+class Vehicle(BaseModel):
+    number_plate = models.CharField(max_length=20, unique=True)
+    category = models.CharField(max_length=50)
```

Comment:
Hi again, can we all discuss what the business value is of having a `category` attribute as a field on the `Slack` channel?